### DISCUSSION TITLE
98

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#45] [General]
# 潮流周刊第98期 - 潮流建筑

周末早上去买菜的路上，看到住的旁边这栋大楼居然给建起来了，在阳光下的无滤镜效果真好看。

https://weekly.tw93.fun/posts/98-%E6%BD%AE%E6%B5%81%E5%BB%BA%E7%AD%91

**Comment by yzh990918** [comment]
web-clipper挺好用的
